python/lsst/ip/diffim/subtractImages.py

```diff
@@ -309,13 +329,12 @@ def run(self, template, science, sources, finalizedPsfApCorrCatalog=None):
             self.log.info("Science variance scaling factor: %.2f", sciVarFactor)
             self.metadata.add("scaleScienceVarianceFactor", sciVarFactor)
 
-        kernelSources = self.makeKernel.selectKernelSources(template, science,
-                                                            candidateList=sources,
-                                                            preconvolved=False)
+        selectSources = self._sourceSelector(sources)
+        self.log.info("%i sources used out of %i from the input catalog", len(selectSources), len(sources))
```

Comment:
Would it make sense to do some alerting/error raising if zero sources are selected? Might save time later...